uxarray/core/dataarray.py

```diff
@@ -429,6 +434,76 @@ def integrate(
 
         return uxda
 
+    def zonal_mean(self, lat=(-90, 90, 10)):
+        """Compute the average along one or more lines of constant latitude.
+
+        The candidate faces at each line of constant latitude are determined using a robust algorithm that treats each edge
+        as a great circle arc, ensuring that extremes along those arcs are considered.
```

Comment:
We don't need to emphasize the "extreme" case specifically. We can just say this robust algorithm takes care of  the intersection point calculation accurate enough

Reply:
Sounds good. Once you have published your work, we can reference the algorithm in your paper in the future for more details.